src/main/MQ2ImGuiConsole.cpp

```diff
@@ -68,12 +68,18 @@ static const int s_userColorCommandLink = USERCOLOR_DIALOG_LINK;
 static const int s_userColorFactionLink = USERCOLOR_FACTION_LINK;
 #endif
 
+// Some default font size
+static const int s_defaultFontSize = 13;
+static const int s_defaultFontIndex = 2;
```

Comment:
What happens if we add more font sizes? default font index is redundant if we also have a default value.